src/test/resources/features/SalesForceLeads.feature

```diff
@@ -1,25 +1,67 @@
 Feature: Salesforce Leads
 
-  @Leads
+
   @salesforce @Leads @Web
   Scenario: Create Leads with Open Status
   and Change Lead Status to Contacted or
   Close Leads if not required to Convert
+  Create New or  Select Existing Accounts, Contacts, and Opportunities
+  Change Opportunity Status Qualifcation to Proposal/Quote
+  Create and Link Quote with Opportunity, Account, Contact
+  Quote  Edit Lines (Select Add Discount), (Add Product)
+  Verify Product Added in the Opportunity
+  Change Opportunity Status to Close  Won after Quote Approval
+  Edit Approved Quote to check mark as Ordered & Create Contract
+  Verify Ordered and Contract created after Quote checked as Ordered
+  Activate Order (Change Status Draft to Activate )
+
+
     Given navigate to Salesforce as "Leads Admin"
     And login to Salesforce with "Leads Admin" credentials
```

Comment:
change here is you want to login with system admin, don't modify the existing code

Reply:
Got it